mappings/net/minecraft/block/Block.mapping

```diff
@@ -488,14 +488,14 @@ CLASS bgs net/minecraft/block/Block
 	METHOD o getStateFactory ()Lbpn;
 	METHOD o (Lbpm;)Z
 		ARG 1 state
-	METHOD o (Lbpm;Lbbb;Let;)I
+	METHOD o getLightSubtracted (Lbpm;Lbbb;Let;)I
```

Comment:
Iirc `darkness` is used in other places for similar function (specifically `World` operations). Should this be made consistent?